ui/src/routes/_layout/organizations/$orgId/products/$productId/repositories/$repoId/create-run.tsx

```diff
@@ -188,7 +188,7 @@ const CreateRunPage = () => {
         notifierRules: '',
         resolutionsFile: '',
         mail: {
-          recipientAddresses: [{}],
+          recipientAddresses: [{ email: '' }],
```

Comment:
Unlike mentioned in the commit message, the original commit is not empty string array, but empty object array with empty objects. Would this rather work with empty array like `[]`, rather than initializing (and always submitting?) a notifier request to `email: ''`?